coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Documentation comments extraction, independently of language or docstyle.
+
+Each of the functions is built upon one another, and at the last,
+exposes a single public function `extract_documentation_with_markers`
+which is used by `DocBaseClass`, to extract documentation.
```

Comment:
also you may want to add this as a reference, see here https://stackoverflow.com/a/7754150/3578554